src/OpenTelemetry.Exporter.Geneva/Internal/MsgPack/MsgPackTraceExporter.cs

```diff
@@ -7,21 +7,24 @@
 using System.Diagnostics;
 using System.Globalization;
 using System.Runtime.InteropServices;
+using System.Text;
 using OpenTelemetry.Exporter.Geneva.Transports;
 using OpenTelemetry.Internal;
 
 namespace OpenTelemetry.Exporter.Geneva.MsgPack;
 
 internal sealed class MsgPackTraceExporter : MsgPackExporter, IDisposable
 {
+    internal const string HTTP_METHOD_V1 = "http.method";
```

Comment:
We will only support stable HTTP conventions. Adding both unstable and stable conventions makes the logic more complex, and I strongly believe there is no need to support old semantics for server httpUrl, as we have never supported it.

Reply:
Removed the unstable part.